Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -776,11 +774,44 @@ internal void OnNetworkDestroy()
             m_IsServer = false;
         }
 
+#region VisList
+        internal void AddToVisList(NetworkIdentity identity)
+        {
+            visList.Add(identity);
+
+            // spawn identity for this conn
+            NetworkServer.ShowForConnection(identity, connectionToClient);
+        }
+
+        internal void RemoveFromVisList(NetworkIdentity identity, bool isDestroyed)
+        {
+            visList.Remove(identity);
+
+            if (!isDestroyed)
+            {
+                // hide identity for this conn
+                NetworkServer.HideForConnection(identity, connectionToClient);
+            }
+        }
+
+        internal void RemoveFromObservers()
+        {
+            foreach (NetworkIdentity identity in visList)
+            {
+                identity.observers.Remove(connectionToClient.connectionId);
+            }
+            visList.Clear();
+        }
+#endregion VisList
+
         internal void ClearObservers()
         {
             foreach (NetworkConnection conn in observers.Values)
             {
-                conn.RemoveFromVisList(this, true);
+                if (conn.playerController != null)
+                {
+                    conn.playerController.RemoveFromVisList(this, true);
```

Comment:
```suggestion
                    conn.playerController?.RemoveFromVisList(this, true);
```

Remove the if,  use null coalescing operator